backend/db.js

```diff
@@ -1,47 +1,24 @@
-const Sequelize = require("sequelize");
+const sqlite3 = require('sqlite3').verbose();
```

Comment:
Just wanted to mention we have been using https://github.com/WiseLibs/better-sqlite3 in other places where we're using sql. 

Don't necessarily need to change things here, but if you want to keep it consistent, it's convenient b/c the API is synchronous

Reply:
I went ahead and swapped them out